src/leaderboard/leaderboard_resource.rs

```diff
@@ -103,6 +106,15 @@ impl Leaderboard {
         request.send(self).await
     }
 
+    pub async fn get_competition_rank(
+        &self,
+        ids: impl IntoIterator<Item = u32>,
+        rank: Option<Order>,
```

Comment:
Could we remove the order here to be in line with `get_rank` (which doesn't take the order but can take it if the caller builds a `GetRankRequest` and calls `send`)? The rest of the PR is green from me.